### ISSUE TITLE
add section for a previously searched images populating into a carousel

### ISSUE THREAD

**Issue by QuantumPhilosophy** [QuantumPhilosophy/anonymous-pumpkin#10]
### User Story
**Landing Page:** User is presented with a carousel of previously searched for images

### Tasks
- [x] Add HTML section for previously viewed images carousel
- [ ] Add a function to populate previously saved images to the carousel

## Notes
Keep in mind while coding the following:
  * **Nice-To-Have:** #15 showing only images searched by user

## Issue Links
**Depends On:** #4 start a Firebase project and initialize in Javascript - Complete
